sesame/src/test/scala/org/w3/banana/sesame/SesameTransactorTest.scala

```diff
@@ -0,0 +1,46 @@
+package org.w3.banana
+package sesame
+
+import org.openrdf.model.Statement
+import org.openrdf.repository.sail.SailRepository
+import org.openrdf.repository.{RepositoryConnection, RepositoryResult}
+import org.openrdf.sail.memory.MemoryStore
+import org.w3.banana.sesame.Sesame._
+import org.w3.banana.util.tryInstances._
+
+import scala.util.Try
+
+class SesameTransactorTest extends TransactorTest[Sesame,Try,RepositoryConnection] {
+
+  val repo = new SailRepository(new MemoryStore)
+  repo.initialize()
+
+  def newConnection() = repo.getConnection
+
+  protected def successfulWrite(con: RepositoryConnection) = transactor.rw(con,con.add(this.expectedGraph))
+
+  protected def successfulRead(con: RepositoryConnection): Try[Sesame#Graph] = {
+    transactor.r(con,{
+      val statements: RepositoryResult[Statement] = con.getStatements(betehess, null, null, true, null)
+      import org.w3.banana.sesame.extensions._
+      ops.makeGraph(statements.toIterable)
+    })
+  }
+
```

Comment:
Unfortunately, it is impossible to do. In rdftest suite I have no knowledge of methods avaliable in connections, the only thing that I know there is just that I have something called "Connection". That is the reason why I added several abstract methods to be implemented in Sesame and Jena Tests that inherit from it. I provided only sesame Jena implementation because I do not know Jena (the last time I used Jena was >1.5 ago), but I think it is not hard to do it.